timur/jest.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  globals: {
+    fetch: require('node-fetch'),
+    CONFIG: {
+      project_name: 'labors'
+    }
+  },
+  testURL: 'http://www.fake.com',
+  transformIgnorePatterns: ['node_modules/(?!(etna-js)/)'],
+  moduleNameMapper: {
+    '^service-worker-loader!': '<rootDir>/__mocks__/service-worker-loader.js',
+    '^.*[.](css|CSS)$': 'identity-obj-proxy',
+    '^react$': '<rootDir>/node_modules/react'
```

Comment:
Pulled the `jest` config into its own file per current recommended practices. Included an entry in `moduleNameMapper` for `react`, to avoid the "Hooks" error.